ui/app/AppLayouts/Browser/BrowserLayout.qml

```diff
@@ -545,10 +545,12 @@ Rectangle {
                     } else if (request.destination === WebEngineView.NewViewInBackgroundTab) {
                         var backgroundTab = tabs.createEmptyTab(currentWebView.profile);
                         request.openIn(backgroundTab.item);
-                    } else if (request.destination === WebEngineView.NewViewInDialog) {
+                    // Disablign popups temporarily since we need to set that webengineview settings / channel and other proeprties
```

Comment:
```suggestion
                    // Disabling popups temporarily since we need to set that webengineview settings / channel and other properties
```